cluster/addons/e2e-rbac-bindings/random-addon-grabbag.yaml

```diff
@@ -1,20 +0,0 @@
-# TODO remove this
-# currently, the kube-addon-manager is adding lots of pods which all share
-# the system:serviceaccount:kube-system:default identity.  We need to subdivide
-# those service accounts, figure out which ones we're going to make bootstrap roles for
-# and bind those particular roles in the addon yaml itself.  This just gets us started
-apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Creating the toothless binding would be the easiest for now, but I don't see a great path forward for removing it (without re-working how the addon manager works, which wouldn't be a bad thing, but...). I could have a cluster that I originally created at 1.6, and I've kept along with upgrades and now at 1.12 we remove the grabbag-binding, and it breaks me. Any thoughts? I know you aren't a huge fan of the addon manager in general...